core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -162,6 +161,12 @@
 
   public static final String S3A_PREFIX = "s3a://";
 
+
+  /**
+   * VIEWS3URL_PREFIX
+   */
+  public static final String S3URL_PREFIX = "s3a://";
```

Comment:
S3A_PREFIX  and S3URL_PREFIX both have same constant.
Please remove once of them.